### DISCUSSION TITLE
PaddleOCR/latest/ppocr/blog/PP-OCRv4_introduction

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14128] [Q&A]
# PaddleOCR/latest/ppocr/blog/PP-OCRv4_introduction

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/blog/PP-OCRv4_introduction.html

**Comment by chenjun2hao** [comment]
优秀，真的做了很多工作。

**Comment by song-2020** [comment]
为什么v1到v3都发论文了，v4都一年了还没发论文

**Comment by tomchen1000** [comment]
Where can we find the en_PP-OCRv4_det_infer.tar and en_PP-OCRv4_rec_infer.tar ?

**Reply by git9875** [reply]
> Where can we find the en_PP-OCRv4_det_infer.tar and en_PP-OCRv4_rec_infer.tar ?

I found en_PP-OCRv4_rec_infer.tar but not en_PP-OCRv4_det_infer.tar

I downloaded  from this URL, but in my case it doesn't seem to be any better than v3.
https://paddleocr.bj.bcebos.com/PP-OCRv4/english/en_PP-OCRv4_rec_infer.tar
I found that URL within this page: https://deepseek.csdn.net/67d15d0fb8d50678a249eeed.html

I also found it on this page, but the actual link for "en_PP-OCRv4_det_infer" goes to "en_PP-OCRv3_det_infer".
https://blog.csdn.net/qq_42493157/article/details/146211273
 The "en_PP-OCRv4_ref_infer" URL is good though, same as the deepseek.csdn.net server.

I tested this URL based on the pattern, but it didn't work:
https://paddleocr.bj.bcebos.com/PP-OCRv4/english/en_PP-OCRv4_det_infer.tar

Like I said before, I don't think it made much an improvement. Maybe it really needs the en_PP-OCRv4_det_infer model.
